experimental/packages/opentelemetry-instrumentation-grpc/src/types.ts

```diff
@@ -35,3 +35,18 @@ export interface GrpcInstrumentationConfig extends InstrumentationConfig {
     };
   };
 }
+
+/**
+ * Tracks whether this instrumentation emits old experimental,
+ * new stable, or both semantic conventions.
+ *
+ * Enum values chosen such that the enum may be used as a bitmask.
+ */
+export const enum SemconvStability {
```

Comment:
Note: we're planning to extract this to a shared place like instrumentation, which is used by all the instrumentation packages, to avoid having to redefine everywhere. If this PR is ready to go before then we can still merge this and replace with the other function when it's ready.

Reply:
done in https://github.com/open-telemetry/opentelemetry-js/pull/5653/commits/eaa6a4307879d94fd167c11fd1bc1e2c1f9880b5